docs/sources/fundamentals/architecture/memberlist.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Memberlist
+weight: 50
+---
+
+## Overview
+
+`Memberlist` is the default key-value store supported by Loki. For more on the function of key-value stores in Loki, please see [Consistent Hash Rings](rings.md). 
+
+This page will provide a little more detail on memberlist and how to configure it in Loki. 
+
+## What is memberlist?
+
+Memberlist is a key-value store that works in a peculiar way: each component has its own copy of the key-value store and, through the gossip protocol, its copy is sent to other components. You can read more about how the gossip protocol works at [How we're using gossip to improve Cortex and Loki availability](https://grafana.com/blog/2020/03/25/how-were-using-gossip-to-improve-cortex-and-loki-availability/). Once a component receives a key-value store from other components, it will merge the received key-value store with its own store and will send its own copy to other components eventually. After some iterations, the different components will all converge to the same key-value store. 
+
+## Configuring memberlist in your Loki config
+
+Configure `memberlist/join_members` with the list of addresses used to reach other components via service discovery. 
+
+You can configure the `join_members` by using a CLI flag or by defining a section in the Loki configuration YAML. Both works accept the same formats and will have the same output. The CLI flag is `-memberlist.join=nodeA` and can be repeated to specify multiple join members and the YAML key is `memberlist/join_members`:
+```yaml
+memberlist:
+  join_members:
+    - nodeA
+    - nodeB
+```
+
+The `join_members` you list are the nodes used to gossip the different rings used by Loki. A good thing to keep in mind when configuring the `join_members` configuration is that the fewer nodes are specified, the less gossiping is necessary, which reduces the number of messages sent. However, the more nodes specified, the more resilient your gossiping since all these nodes must go get down for your memberlist to stop working.
```

Comment:
```suggestion
The `join_members` you list are the nodes used to gossip the different rings used by Loki. A good thing to keep in mind when configuring the `join_members` configuration is that the fewer nodes are specified, the less gossiping is necessary, which reduces the number of messages sent. However, the more nodes specified, the more resilient your gossiping since all these nodes must go down for your memberlist to stop working.
```

Reply:
not sure if my suggestion is right but my bad English says this was a mistake? :D ^^